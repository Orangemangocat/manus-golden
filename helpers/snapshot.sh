#!/usr/bin/env bash
set -euo pipefail
OUT="/var/lib/manus/snapshots/sys_$(date +%F_%H%M%S).txt"
{
  uname -a
  df -h
  free -m
  docker ps
  uptime
  systemctl --failed || true
} > "$OUT"
/usr/local/bin/proof snapshot "$OUT"
echo "$OUT"
