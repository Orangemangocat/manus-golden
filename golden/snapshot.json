{
  "task": "snapshot",
  "description": "Capture system status and write proof.",
  "steps": [
    {"id":"snap","cmd":"/usr/local/bin/snapshot.sh"}
  ]
}
